fastn-update/src/lib.rs

```diff
@@ -16,6 +16,12 @@ pub enum ManifestError {
         #[source]
         source: serde_json::Error,
     },
+    #[error("Failed to read manifest content for package '{package}'")]
+    DsReadError {
+        package: String,
```

Comment:
This is a bad pattern. Store the `package` in the error that stores `ManifestError`.

Reply:
If you do this you can not use `#[from]` and create auto converters.

Reply:
snafu solves this better: https://docs.rs/snafu/latest/snafu/trait.ResultExt.html#tymethod.context